packages/api/models/rules.js

```diff
@@ -159,16 +159,25 @@ class Rule extends Manager {
           return (mapping.EventSourceArn === item.rule.value);
         });
       if (mappingExists) {
-        item.rule.arn = mappingExists.UUID;
-        return item;
+        const mappingEnabled = listData.EventSourceMappings
```

Comment:
We created the event source mapping with state 'ENABLED'.  Do we know how the state becomes 'not Enabled'?

Reply:
Ok, I was able to reproduce the state:
```     { UUID: '5f0dc194-5cab-4149-a292-e408689779a9',
       BatchSize: 100,
       EventSourceArn: 'arn:aws:kinesis:us-east-1:893015583569:stream/mhs-cumulus-testStream',
       FunctionArn: 'arn:aws:lambda:us-east-1:893015583569:function:mhs-cumulus-kinesisConsumer',
       LastModified: 2018-07-19T19:38:00.000Z,
       LastProcessingResult: 'PROBLEM: Stream not found. Recreate the stream and re-enable the event source mapping to resume processing from TRIM_HORIZON.',
       State: 'Disabled',
       StateTransitionReason: 'Unrecoverable error' },
```